plugins/modules/redfish_config.py

```diff
@@ -124,7 +124,12 @@
     default: {}
     version_added: '5.7.0'
 
-author: "Jose Delarosa (@jose-delarosa)"
```

Comment:
@felixfontein what's the normal practice for listing authors in a module? Is it just the original contributor or anyone actively developing?

Reply:
It depends a lot. Some folks never add themselves as authors, even though they basically rewrite the module, while others want to add themselves when implementing even smaller features. It's basically a matter of taste. I'm not sure that implementing an apparently simple feature like this one legitimates that four new authors should be added. Adding them to BOTMETA as maintainers is one thing, but to the `author` field looks really strange to me. @TSKushal can you please explain how all these authors contributed to this PR?

Reply:
We just thought this was the standard practice followed by everyone to add the author names for any contribution done.
All the authors mentioned here are actively contributing to the open source work of redfish modules from us. If it is inappropriate to add everyone  here for single method/small change, I can remove the others.

Reply:
I'm fine with this. Maybe one day I'll add myself.

Reply:
Hope this is clarified now. Can I resolve it?